_episodes_rmd/06_day5.Rmd

```diff
@@ -0,0 +1,217 @@
+---
+title: "Day 6"
+teaching: 0
+exercises: 0
+questions:
+- "change"
+objectives:
```

Comment:
I know this is still in draft form, but it might be a good to add in the objectives and questions posed in this capstone lesson.